src/app/file-manager/services/gcs-api.service.ts

```diff
@@ -61,18 +61,18 @@ export class GcsApiService extends GcsService {
   public cancelAll() {
     const itemsToDownload = this.getDownloadingFiles();
     const itemsToUpload = this.getUploadingFiles();
-    const itemsToExport = this.getExportPendingFiles();
+    const itemsToExport = this.getExportingFiles();
     if (itemsToDownload.length > 0) {
       this.electronService.ipcRenderer.send('download-cancel');
-      this.cancelItemsStatus(itemsToDownload);
+      this.cancelItemsStatus(Type.DOWNLOAD);
     }
     if (itemsToUpload.length > 0) {
       this.electronService.ipcRenderer.send('upload-cancel');
-      this.cancelItemsStatus(itemsToUpload);
+      this.cancelItemsStatus(Type.UPLOAD);
     }
     if (itemsToExport.length > 0) {
       this.electronService.ipcRenderer.send('export-gcp-cancel');
```

Comment:
This is being sent but I don't see where a handler is registered to receive it.

Reply:
This has been removed.